bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -506,11 +535,26 @@ private void stopSensorPolling() {
         }
     }
 
+    private void startScenePolling() {
+        if (scenePollingJob == null || scenePollingJob.isCancelled()) {
+            scenePollingJob = scheduler.scheduleWithFixedDelay(scenePollingRunnable, 1, scenePollingInterval,
+                    TimeUnit.MILLISECONDS);
```

Comment:
Maybe consider scenePollingIntervalin minutes.
And add a delay to start this job to give a chance to get first the groups.